Squirrel/Controllers/AccountController.cs

```diff
@@ -1,11 +1,17 @@
 using Microsoft.AspNetCore.Mvc;
+using Squirrel.Contexts;
 
 namespace Squirrel.Controllers
 {
     [ApiController]
     [Route("api/[controller]")]
     public class AccountController : Controller
     {
+        private readonly ApplicationContext context;
 
+        public AccountController(ApplicationContext context)
+        {
+            this.context = context;
```

Comment:
`this` word is deprecated and it's better to use underscores by Microsoft convension